userdocs/profiles.dev/docs/tutorial-basics/install-a-profile.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 3
+---
+
+# Install a profile
+
+:::info
+
+This stage of the tutorial assumes you have prepared your environment correctly.
+
+Please refer back to the [set up docs](/docs/tutorial-basics/setup) if not.
+:::
+
+To install a profile, we use `pctl`.
+
+First, take care to ensure that you are in your GitOps repo directory on your local
```

Comment:
local filesystem is redundant and implies we care where the repo is. We don't. :)

Reply:
oh we don't anymore? yay! that is so much better.